README.md

```diff
@@ -160,51 +160,57 @@ $ astro dev --help
 $ astro deploy --help
 ```
 
-## Development
+## Local Development
 
 How to get started as a developer:
 
-1. Build:
+1. Install `go` 1.7+ - https://go.dev/doc/install
```

Comment:
This should be 1.11+ instead of 1.7+ as go mod support was added from 1.11 onwards.